tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -207,5 +209,30 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            DirectoryInfo logsDirInfo = new DirectoryInfo(logsDirectory);
+            FileInfo[] logFiles = logsDirInfo.GetFiles().OrderBy(p => p.LastWriteTime).ToArray();
```

Comment:
You can use the static `Directory.GetFiles()` without creating a `DirectoryInfo` instance:
```suggestion
            FileInfo[] logFiles = Directory.GetFiles().OrderBy(p => p.LastWriteTime).ToArray();
```